qemu/tests/qemu_guest_agent.py

```diff
@@ -2588,6 +2588,162 @@ def bl_check(qga_cmd):
             cmd = "mv -f /etc/sysconfig/qemu-ga-bk /etc/sysconfig/qemu-ga"
             session.cmd(cmd)
 
+    @error_context.context_aware
+    def gagent_check_os_basic_info(self, test, params, env):
+        """
+        Get hostname,timezone,currently active users on the vm and login time.
+        Steps:
+        1) Check host name.
+        2) Check timezone name.
+        3) check timezone's offset to UTS in seconds.
+        4) Check all active users number.
+        5) Check every user info.
+        6) Check every user's domain(windows only)
+        7) Get the earlier loggin time for the same user.
+        8) Check the login time for every user.
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        session = self._get_session(params, None)
+        self._open_session_list.append(session)
+
+        def _result_check(rsult_qga, rsult_guest):
+            if rsult_qga != rsult_guest:
+                msg = "The result is different between qga and guest\n"
+                msg += "from qga: %s\n" % rsult_qga
+                msg += "from guest: %s\n" % rsult_guest
+                test.fail(msg)
+
+        error_context.context("Check host name of guest.", logging.info)
+        host_name_ga = self.gagent.get_host_name()["host-name"]
+        cmd_get_host_name = params["cmd_get_host_name"]
+        host_name_guest = session.cmd_output(cmd_get_host_name).strip()
+        _result_check(host_name_ga, host_name_guest)
+
+        error_context.context("Check timezone of guest.", logging.info)
+        timezone_ga = self.gagent.get_timezone()
+        timezone_name_ga = timezone_ga["zone"]
+        timezone_offset_ga = timezone_ga["offset"]
+
+        logging.info("Check timezone name.")
+        cmd_get_timezone_name = params["cmd_get_timezone_name"]
+        timezone_name_guest = session.cmd_output(cmd_get_timezone_name).\
+            strip()
+        if params["os_type"] == "windows":
+            # there are standard name and daylight name for windows os
+            # both are accepted.
+            cmd_dlight_name = params["cmd_get_timezone_dlight_name"]
+            timezone_dlight_name_guest = session.cmd_output(cmd_dlight_name).\
```

Comment:
for line break like this, please do not use a backslash like this, better to
```python
        timezone_dlight_name_guest = session.cmd_output(
            cmd_dlight_name).strip()
```
quote from [PEP8](https://www.python.org/dev/peps/pep-0008/)
```
The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces.
```

Reply:
ok, modified.